installing/disconnected_install/installing-mirroring-creating-registry.adoc

```diff
@@ -14,7 +14,7 @@ If you already have a container image registry, such as Red Hat Quay, you can sk
 == Prerequisites
 
 * An {product-title} subscription.
-* {op-system-base-full} 8 with Podman 3.3 installed.
+* {op-system-base-full} 8 with Podman 3.3 and openssl installed.
```

Comment:
It seems that we use `openssl` vs `OpenSSL` to an alarming degree. However, I think the [latter is correct](https://www.openssl.org/).